### ISSUE TITLE
Update the glossary to the latest network state

### ISSUE THREAD

**Issue by mhluongo** [keep-network/keep-core#1540]
Include

* all language around staking roles
* all language around upgrades
* any language found in the token dashboard

Remove

* any references to the outdated whitepaper, or terms that aren't in active use in our codebase today

Supersedes #1032

**Comment by mhluongo** [comment]
Yes, I'm adding everyone under the sun to look at this because I can't decided :joy:

**Comment by lionakhnazarov** [comment]
Closing as stale. Legacy community/docs task no longer reflecting the current repo; re-file against current main if still wanted.
